lib/cli.js

```diff
@@ -124,9 +125,14 @@ exports.run = () => {
 	if (cli.flags.tap && !cli.flags.watch) {
 		reporter = tapReporter();
 	} else if (cli.flags.verbose || isCi) {
-		reporter = verboseReporter();
+		reporter = verboseReporter({
+			disableColors: cli.flags.color !== undefined
```

Comment:
Without looking into `meow`, I'm surprised this isn't a boolean. Does it become `false` if `--no-color` is set? Comparing against `false` seems better than comparing against `undefined`.